sentry-android-replay/src/main/java/io/sentry/android/replay/capture/BaseCaptureStrategy.kt

```diff
@@ -180,7 +191,32 @@ internal abstract class BaseCaptureStrategy(
                     val breadcrumbCategory: String?
                     val breadcrumbData = mutableMapOf<String, Any?>()
                     when {
-                        breadcrumb.category == "http" -> return@forEach
+                        breadcrumb.category == "http" -> {
+                            if (!breadcrumb.isValidForRRWebSpan()) {
+                                return@forEach
+                            }
+                            recordingPayload += RRWebSpanEvent().apply {
+                                timestamp = breadcrumb.timestamp.time
+                                op = "resource.http"
+                                description = breadcrumb.data["url"] as String
+                                startTimestamp =
+                                    (breadcrumb.data["start_timestamp"] as Long) / 1000.0
+                                this.endTimestamp =
```

Comment:
would it make sense to factor this out into a separate method to avoid `endTimestamp` vs `this.endTimestamp`?